README.md

```diff
@@ -6,13 +6,38 @@
 
 ![ORAS](https://github.com/oras-project/oras-www/raw/main/docs/assets/images/oras.png)
 
+## Project status
+
+### v1: stable
+
+As there are various stable projects depending on the ORAS Go library, the
+[`v1`](https://github.com/oras-project/oras-go/tree/v1) branch
+is maintained for API stability, dependency updates, and security patches.
+All `v1.*` releases are based upon this branch.
+
+If you are seeking stability over new features, you are highly encouraged
+to use releases with major version `1`.
+
+### v2: experimental
+
+In contrast to the `v1` branch, the
+[`main`](https://github.com/oras-project/oras-go/tree/main) branch
+is where all new feature development will occur. Since ORAS is a
+primary staging ground for the
+[ORAS Artifacts Specification](https://github.com/oras-project/artifacts-spec),
+changes are expected to occur regularly to meet new requirements.
+This will likely introduce backward-incompatible changes, resulting in
```

Comment:
maybe make this sound a little less scary by adding may, and will follow semver semantics with a 2.* release for consumers to opt in?

Reply:
Good idea, I added a code suggestion for this below 😄 How does that sound?